lib/classes/manage_location.rb

```diff
@@ -0,0 +1,20 @@
+class ManageLocation
+  def initialize(location_id)
+    @location = Location.find_by(id: location_id)
+  end
+
+  def empty_location?
+    @location.packages_locations.count.zero? &&
+      @location.package_types.count.zero? &&
+      PackagesInventory.where(location_id: @location.id).count.zero? &&
+      Printer.where(location_id: @location.id).count.zero? &&
+      Stocktake.where(location_id: @location.id).count.zero?
+  end
+
+  def self.merge_location(source_location, destination_location)
+    %w[Printer Stocktake PackagesInventory PackageType PackagesLocation Package].each do |klass|
```

Comment:
Do packages have a `location_id` ? If so we should probably remove that column

Reply:
Yeah it is there, maybe an old column as it is not used anywhere. But just to be on the safer side updating that value too.

Reply:
yup sounds good, I'll create a ticket for removing this column separately